subsys/bluetooth/services/ots/ots.c

```diff
@@ -442,6 +442,22 @@ void *bt_ots_svc_decl_get(struct bt_ots *ots)
 }
 #endif
 
+static void oacp_indicate_work_handler(struct k_work *work)
+{
+	struct bt_gatt_ots_indicate *ind = CONTAINER_OF(work, struct bt_gatt_ots_indicate, work);
+	struct bt_ots *ots = CONTAINER_OF(ind, struct bt_ots, oacp_ind);
+
+	bt_gatt_indicate(NULL, &ots->oacp_ind.params);
+}
+
+static void olcp_indicate_work_handler(struct k_work *work)
+{
+	struct bt_gatt_ots_indicate *ind = CONTAINER_OF(work, struct bt_gatt_ots_indicate, work);
+	struct bt_ots *ots = CONTAINER_OF(ind, struct bt_ots, olcp_ind);
+
+	bt_gatt_indicate(NULL, &ots->olcp_ind.params);
+}
```

Comment:
Since delaying indications does guard us from new request, we might need to prevent any OACP or OLCP requests while these are pending. 

If we get a new request in before these are handled (not unlikely if EATT is enabled), then we will overwrite these, and not send the previous ones, right?

Reply:
OK, I've added check and since spec doesn't mention on how this should be handled I used BT_ATT_ERR_PROCEDURE_IN_PROGRESS (0xFE) error code.